.env.example

```diff
@@ -0,0 +1,4 @@
+VITE_TOKEN_SERVER_URL=/api
+VITE_LOCAL_SERVER_URL=https://your-ip:port/
```

Comment:
This setting was only used because was needed for renderRedirectToMobile, it can safely be removed. (i think)